server/core/src/main/java/org/infinispan/server/core/configuration/ProtocolServerConfiguration.java

```diff
@@ -33,14 +35,25 @@ public abstract class ProtocolServerConfiguration<T extends ProtocolServerConfig
    public static final AttributeDefinition<Boolean> ZERO_CAPACITY_NODE = AttributeDefinition.builder(Attribute.ZERO_CAPACITY_NODE, false).immutable().build();
    public static final AttributeDefinition<String> SOCKET_BINDING = AttributeDefinition.builder(Attribute.SOCKET_BINDING, null, String.class).immutable().build();
    public static final AttributeDefinition<Boolean> IMPLICIT_CONNECTOR = AttributeDefinition.builder("implicit-connector", false).immutable().autoPersist(false).build();
+   public static final AttributeDefinition<String> MAX_CONTENT_LENGTH = AttributeDefinition.builder("max-content-length", "10MB", String.class)
```

Comment:
Use a constant in `org.infinispan.server.core.configuration.Attribute`

Reply:
I will consolidate this like the max content length attribute